docs/execution-analysis/gpu_acceleration.md

```diff
@@ -1,29 +1,59 @@
+{% hint style="info" %}
+This version is not available on [pypi.org](pypi.org/project/concrete-python), which only hosts wheels with CPU support. 
+{% endhint %}
+
+To use GPU acceleration, install the GPU/CUDA wheel from our [Zama public PyPI repository] (https://pypi.zama.ai) using the following command:
 
 `pip install concrete-python  --index-url https://pypi.zama.ai/gpu`.
 
-Once a GPU/CUDA flavor python wheel is installed, FHE program compilation must be [configured](../guides/configure.md) using the **use_gpu** option to enable GPU offloading.
+After installing the GPU/CUDA wheel, you must [configure] ((../guides/configure.md)) the FHE program compilation to enable GPU offloading using the `use_gpu` option.
 
 {% hint style="info" %}
-Our GPU wheels are built with CUDA 11.8 and should be compatible with higher versions of CUDA.
+Our GPU wheels are built with CUDA 11.8 and are compatible with higher versions of CUDA.
 {% endhint %}
 
 ## GPU execution configuration
 
-By default the compiler and runtime will make use of all resources available on the system, to include all CPU cores and GPUs. This can be adjusted by using environment variables.
-The following variables are relevant in this context:
-
-* **SDFG_NUM_THREADS**: Integer = Number of hardware threads on the system, including hyperthreading, less number of GPUs in use.
-  * Number of CPU threads to execute concurrently to GPU for workloads that can be offloaded. As GPU scheduler threads (including CUDA threads and those used within Concrete) are necessary and can be a bottleneck or interfere with worker thread execution, it is recommended to undersubscribe the CPU hardware threads by the number of GPU devices used.
-* **SDFG_NUM_GPUS**: Integer = Number of GPUs available
-  * Number of GPUs to use for offloading. This can be set at any value between 1 and the total number of GPUs on the system.
-* **SDFG_MAX_BATCH_SIZE**: Integer = LLONG_MAX (no batch size limit)
-  * Limit the maximum batch size to offload in cases where the GPU memory is insufficient.
-* **SDFG_DEVICE_TO_CORE_RATIO**: Integer = Ratio between the compute capability of the GPU (at index 0) and a CPU core
-  * Ratio between GPU and CPU used to balance the load between CPU and GPU. If the GPU is starved, this can be set at higher values to increase the amount of work offloaded.
-* **OMP_NUM_THREADS**: Integer = Number of hardware threads on the system, including hyperthreading
-  * Portions of program execution that are not yet supported for GPU offload are parallelized using OpenMP on the CPU.
+By default the compiler and runtime will use all available system resources, including all CPU cores and GPUs. You can adjust this by using the following environment variables:
+
+### SDFG_NUM_THREADS
+- **Type**: Integer
+- **Description:** The number of hardware threads on the system (including hyperthreading) minus the number of GPUs in use.
```

Comment:
The notation is used in other parts of the doc - might have been changed - but the "description" is actually the default value. These are environment variables that can be customised by the user. 

If the notation of "xxx: type = default_value" is dropped throughout the doc I would at least replace description with default.

Reply:
Do you mean like this?
### SDFG_NUM_THREADS
- **Type**: Integer
- **Default value:** The number of hardware threads on the system (including hyperthreading) minus the number of GPUs in use.

Reply:
Yes, if this is unified across the doc, otherwise it would be good to keep things consistent.

Reply:
For example https://docs.zama.ai/concrete/guides/configure uses the same notation, so unless it is changed across all doc pages I think it would be best to have a single pattern.

Reply:
Agree for consistency. Let's decide which form we should adopt across all docs ( I can modify the rest accordingly in a seperate PR). From a developer's view, which way is easier to read for you guys?

---

**SDFG_NUM_THREADS**: `Integer` = The number of hardware threads on the system (including hyperthreading) minus the number of GPUs in use.

---

#### SDFG_NUM_THREADS
**Type**: Integer
**Default value**: The number of hardware threads on the system (including hyperthreading) minus the number of GPUs in use.

Reply:
cc @bcm-at-zama and @BourgerieQuentin any preference here?

Reply:
no pref

Reply:
We should always include the same attributes for parameters when documenting them, I would recommend the following formats

---

Parameters with long description and explanations, like here

#### SDFG_NUM_THREADS
**Type**: Integer
**Default value**: The number of hardware threads on the system (including hyperthreading) minus the number of GPUs in use.
**Description**:This variable determines the number of CPU threads that execute in paralelle with the GPU for offloadable workloads. GPU scheduler threads (including CUDA threads and those used within Concrete) are necessary but can block or interfere with worker thread execution. Therefore, it is recommended to undersubscribe the CPU hardware threads by the number of GPU devices used.
**Required**: No

--- 

Parameters with short descriptions such as in  https://docs.zama.ai/concrete/guides/configure , we can combine them in table format. 
**Parameters:**
| Name      | Type     | Default value | Required | Description                        |
|-----------|----------|----------|---------|------------------------------------|
| param1    | Integer      | 0      | Yes    |xxx    |
| param2    | Bolean      | False      | No    |xxx    |

This is very easy to read and search. 
@antoniupop Wdyt?